glyphs-reader/src/glyphdata.rs

```diff
@@ -85,11 +126,170 @@ impl GlyphData {
     }
 
     /// Look up info for a glyph by codepoint
-    pub fn get_by_codepoint(&self, codepoint: u32) -> Option<&GlyphInfo> {
+    fn get_by_codepoint(&self, codepoint: u32) -> Option<&GlyphInfo> {
         self.unicode_map
             .get(&codepoint)
             .and_then(|idx| self.data.get(*idx as usize))
     }
+
+    // https://github.com/googlefonts/glyphsLib/blob/e2ebf5b517d/Lib/glyphsLib/glyphdata.py#L199
+    fn construct_category(&self, name: &str) -> Option<(Category, Subcategory)> {
+        if name.starts_with('_') {
```

Comment:
a few comments would help. I know I can read the python but ... ideally I wouldn't have to. E.g. what is the signiicance of starting with _? Maybe we could have a function, if is_something(name) would be clearer.

Reply:
glyph names that start with an underscore are usually used as "parts" to be combined in other composite glyphs, and are set as non-export by default, see https://glyphsapp.com/learn/smart-components, section "Setting up a Smart Glyph":

> enter a name that starts with _part, because it is a part we can reuse in other glyphs. Also, glyphs created with an underscore at the beginning will be set to non-exporting by default, which makes sense for us.